inc/Config/QueryRunner/QueryRunner.php

```diff
@@ -82,7 +82,12 @@ protected function get_raw_response_data( array $input_variables ): array|WP_Err
 			RequestOptions::JSON    => $body,
 		];
 
-		$this->http_client->init( $endpoint_base );
+		$client_options = [
+			// This is a custom option ignored by GuzzleHttp\Client, so it is prefixed with '__'.
+			'__default_cache_ttl' => $this->query_context->get_cache_ttl( $input_variables ),
```

Comment:
Would be nice to stick this on a class as a constant, maybe on `HttpClient` where it is used.